db/migrations/manual/20211229-runout.sql

```diff
@@ -0,0 +1,21 @@
+-- This is a manual DB wrangle to convert Dec CCLF attribution into runouts
+-- We will undo this and convert back to Dec CCLF once the runout for 2022 is received
+
+BEGIN;
+DO $$
+
+DECLARE cclfids integer ARRAY;
+BEGIN
+cclfids := ARRAY(SELECT id FROM cclf_files WHERE timestamp > '2021-11-30' AND aco_cms_id NOT LIKE ALL(ARRAY['A999%', 'V99%', 'E999%']);
```

Comment:
you could just include `WHERE type = 0` just in case.

Reply:
Though down the migration wouldn't be able to discern between the two if any existed.